templates/security/notices/usn.html

```diff
@@ -4,39 +4,50 @@
 
 {% block content %}
 <section class="p-strip--suru-topped">
+  <div class="u-fixed-width">
+    <h1>{{ notice.id }}: {{ notice.title }}</h1>
+    <p class="p-muted-heading">{{ notice.published }}</p>
+    <p>{{ notice.summary | safe }}</p>
+    <div class="col-5 u-hide--large">
+      <div class="p-card--highlighted">
+        <h3 class="p-card__title">Reduce your security exposure</h3>
+        <p>Ubuntu Pro provides ten-year security coverage to 23,000+ packages in Main and Universe, and it is free for up to five machines.</p>
```

Comment:
Change this text to "Ubuntu Pro provides ten-year security coverage to 25,000+packages in Main and Universe repositories, and it is free for up to five machines." as requested by @lechsandecki